### PR TITLE
Fix pre release issues

### PR DESCRIPTION
**Pull request by joshcanhelp**
- Remove beta plugin (#400) 
- Fix shorthand array
- `die` -> `exit` for consistency
- `parse_url()` -> `wp_parse_url()` as recommended by WordPress

### CODE REVIEW

File: WP_Auth0.php [hunk 1]
```diff
@@ -541,9 +541,9 @@ function get_auth0_curatedBlogName() {
 
     // WordPress can have a blank site title, which will cause initial client creation to fail
     if ( empty( $name ) ) {
-	    $name = parse_url( home_url(), PHP_URL_HOST );
+	    $name = wp_parse_url( home_url(), PHP_URL_HOST );
```
Comment (joshcanhelp) [inline WP_Auth0.php]: `wp_parse_url()` is a safer alternative recommended by WordPress

File: lib/WP_Auth0_Amplificator.php [hunk 2]
```diff
@@ -16,7 +16,7 @@ public function init() {
 
 	public function share() {
 		if ( ! isset( $_POST['provider'] ) ) {
-			wp_die();
```
Comment (joshcanhelp) [inline lib/WP_Auth0_Amplificator.php]: Not a good way to quit an AJAX process

File: lib/admin/WP_Auth0_Admin.php [hunk 3]
```diff
@@ -209,7 +209,7 @@ public function init() {
 	 * Enqueue scripts for all Auth0 wp-admin pages
 	 */
 	public function admin_enqueue() {
-		$wpa0_pages     = [ 'wpa0', 'wpa0-errors', 'wpa0-users-export', 'wpa0-import-settings', 'wpa0-setup' ];
+		$wpa0_pages     = array( 'wpa0', 'wpa0-errors', 'wpa0-users-export', 'wpa0-import-settings', 'wpa0-setup' );
```
Comment (joshcanhelp) [inline lib/admin/WP_Auth0_Admin.php]: PHP 5.3 does not support shorthand arrays

File: templates/a0-widget-setup-form.php [hunk 4]
```diff
@@ -5,7 +5,6 @@
 $form_title = isset( $instance[ 'form_title' ] ) ? $instance[ 'form_title' ] : '';
 $social_big_buttons = isset( $instance[ 'social_big_buttons' ] ) ? $instance[ 'social_big_buttons' ] : '';
 $gravatar = isset( $instance[ 'gravatar' ] ) ? $instance[ 'gravatar' ] : '';
-$show_icon = isset( $instance[ 'show_icon' ] ) ? $instance[ 'show_icon' ] : '';
```
Comment (joshcanhelp) [inline templates/a0-widget-setup-form.php]: Unused option

File: WP_Auth0.php [hunk 5]
```diff
@@ -558,3 +558,11 @@ function get_auth0_curatedBlogName() {
 
 $a0_plugin = new WP_Auth0();
 $a0_plugin->init();
+
+/*
+ * Beta plugin deactivation
+ */
+
+// Passwordless beta testing - https://github.com/auth0/wp-auth0/issues/400
+remove_filter( 'login_message', 'wp_auth0_pwl_plugin_login_message_before', 5 );
```
Comment (joshcanhelp) [inline WP_Auth0.php]: Deactivating a beta plugin released earlier

File: lib/WP_Auth0_DBManager.php [hunk 6]
```diff
@@ -113,13 +113,7 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 			$options->set('cache_expiration', 1440);
 
 			// Update Client
-			if (!empty($client_id) && !empty($domain)) {
-				$payload = array(
```
Comment (joshcanhelp) [inline lib/WP_Auth0_DBManager.php]: This was overwriting existing Application settings, which could have unintended effects.

File: lib/WP_Auth0_DBManager.php [hunk 7]
```diff
@@ -164,15 +158,6 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 
 				$payload = array(
 					'app_type' => 'regular_web',
-					'callbacks' => array(
```
Comment (joshcanhelp) [inline lib/WP_Auth0_DBManager.php]: This was overwriting existing Application settings, which could have unintended effects.

File: templates/a0-widget-setup-form.php (hunk 4 above)
Comment (lbalmaceda) [reply templates/a0-widget-setup-form.php]: could this be considered a breaking change?

File: lib/initial-setup/WP_Auth0_InitialSetup_Connections.php [hunk 8]
```diff
@@ -191,7 +191,7 @@ public function callback() {
       try {
         $input = $operations->social_validation( $this->a0_options->get( 'auth0_app_token' ), $old_input, $input, $provider_name, $provider['options'] );
       } catch ( Exception $e ) {
-        die( $e->getMessage() );
+        exit( $e->getMessage() );
```
Comment (lbalmaceda) [inline lib/initial-setup/WP_Auth0_InitialSetup_Connections.php]: `die` `wp_die` and `exit` are analogous ?

Comment (joshcanhelp) [reply lib/initial-setup/WP_Auth0_InitialSetup_Connections.php]: yes

File: templates/a0-widget-setup-form.php (hunk 4 above)
Comment (joshcanhelp) [reply templates/a0-widget-setup-form.php]: No, not used at all in the file or anywhere else.

File: lib/initial-setup/WP_Auth0_InitialSetup_Connections.php (hunk 8 above)
Comment (joshcanhelp) [reply lib/initial-setup/WP_Auth0_InitialSetup_Connections.php]: `die` and `exit` are, `wp_die` ends the process with HTML output and a 500 error response.

**Pull request merged**